logbook/handlers.py

```diff
@@ -1391,14 +1391,26 @@ def __init__(self, application_name=None, address=None,
             else:
                 address = '/dev/log'
 
-        self.address = address
+        self.remote_address = self.address = address
```

Comment:
Why is `remote_address` needed? Why not just use `address`?

Reply:
This is explained in the commit message for b9daef7311f4816467db8af2b7573bad1bb55c33.